fcq/models.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+
+from classes.models import Department
+
+
+# Proposed changes:
+"""
+have course rating, times taught, intructor
+rating, and course challenge in the FCQ Table
+instead of being array fields in the Teacher
+Table. 
+
+Foreign key back to the professor which coresponds
+with the FCQ.
+
+Change department and course to Foreign Keys
+to their respective models
+(this is dependent on the database changes
+happening for the class models, see notes there.
+If that doesn't happen, leave these as Chars)
+
+"""
```

Comment:
I agree with the first change, that way if we remove or add fcqs the averages for the professors change as we would then calculate it.

I'm curious what the use case for having a foreign key back to the professor is, I would say that change makes sense though.

And I agree with the class changes, so dept and course should be foreign keys. (notably not section, I don't think that would be feasible or usable)